### PR TITLE
Nat 2017/4/4 neighbourhoods with moved feature geom (27 require PIP)

### PR DESCRIPTION
**Pull request by nvkelso**
This PR is limited to just the 27 features which were POINT only geometries where the label moved, which effectively moved the PIP geometry, requiring a re-PIPing.

### CODE REVIEW

**Comment by nvkelso** [comment]
Please make sure Zach's work and Nat's work aren't stamping on each other here.

**Comment by nvkelso** [comment]
Related: 

- https://github.com/whosonfirst-data/whosonfirst-data/pull/717
- https://github.com/whosonfirst-data/whosonfirst-data/pull/718

File: data/110/871/340/3/1108713403.geojson [hunk 1]
```diff
@@ -6,24 +6,31 @@
...
+    "lbl:longitude":-71.089067,
     "mz:hierarchy_label":1,
     "mz:is_funky":0,
+    "mz:is_hard_boundary":0,
     "mz:is_landuse_aoi":0,
+    "mz:is_official":0,
     "mz:max_zoom":18,
     "mz:min_zoom":12,
+    "mz:tier_metro":1,
+    "name:Kendallsquare)":[
```
Comment (stepps00) [inline data/110/871/340/3/1108713403.geojson]: Looks like the prop and value were switched around here..?

File: data/110/871/340/5/1108713405.geojson [hunk 2]
```diff
@@ -6,24 +6,31 @@
...
+        "eng_x_preferred"
+    ],
     "name:eng_x_variant":[
         "Porter Sq",
         "Portersquare"
     ],
     "reversegeo:latitude":42.388718,
     "reversegeo:longitude":-71.11957,
     "src:geom":"mz",
+    "src:lbl":"mz",
```
Comment (stepps00) [inline data/110/871/340/5/1108713405.geojson]: This isn't wrong, but we have a few different versions of this property. I've started to use `src:lbl:centroid`.

**Comment by stepps00** [comment]
Left comments inline. This _shouldn't_ have an affect on Boston area what that Zach has completed since Zach's Boston area work was merged yesterday.

File: data/110/871/340/3/1108713403.geojson (hunk 1 above)
Comment (nvkelso) [inline data/110/871/340/3/1108713403.geojson]: uhhh, that's weird!

File: data/110/871/340/5/1108713405.geojson (hunk 2 above)
Comment (nvkelso) [inline data/110/871/340/5/1108713405.geojson]: I've updated my script, but would need to re-run and probably submit a new PR (which looks likely anyhow per https://github.com/whosonfirst-data/whosonfirst-data/pull/716#discussion_r109773230).

File: data/110/871/340/5/1108713405.geojson [hunk 3]
```diff
@@ -6,24 +6,31 @@
...
+    "lbl:longitude":-71.119035,
     "mz:hierarchy_label":1,
     "mz:is_funky":0,
+    "mz:is_hard_boundary":0,
     "mz:is_landuse_aoi":0,
+    "mz:is_official":0,
     "mz:max_zoom":18,
     "mz:min_zoom":13,
+    "mz:tier_metro":1,
+    "name:Portersquare)":[
```
Comment (nvkelso) [inline data/110/871/340/5/1108713405.geojson]: Same name namespace problem.

File: data/110/871/340/3/1108713403.geojson (hunk 1 above)
Comment (stepps00) [inline data/110/871/340/3/1108713403.geojson]: (this issue is affecting this PR and the PR with 16k features)

**Comment by nvkelso** [comment]
Closing in favor of #720.

**Pull request closed without merging**